scripts/server_scripts/mock_health_event_publisher.py

```diff
@@ -0,0 +1,248 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error
```

Comment:
Remove disabling import error.

Reply:
Getting multiple codacy issues for imports in this file. Imports are directly from cortx (lines 32 to 38), so these issues are expected.